test/db/cmd/cmd_flags

```diff
@@ -212,7 +212,7 @@ aaa
 flj~{254}
 EOF
 EXPECT=<<EOF
-{"name":"reloc.operator_delete_void","realname":"operator delete(void*)","size":8,"offset":16432}
+{"name":"obj.__dso_handle","realname":"__dso_handle","size":0,"offset":16496}
```

Comment:
This is wrong - please change the index in `flj` to preserve the same symbol - please check what this test is about.